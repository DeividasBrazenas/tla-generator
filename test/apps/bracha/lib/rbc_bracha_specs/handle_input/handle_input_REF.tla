---------------------- MODULE handle_input_REF ----------------------
EXTENDS handle_input

(*
    1. Result should exist for the broadcaster node.
    2. propose_sent should be set to TRUE in the broadcaster node.
    3. propose message should be sent to all peers from the broadcaster node.
*)
IsProposeSent(res, node) ==   
    /\ res[node] /= NULL
    /\ res[node][2].propose_sent = TRUE
    /\ \A peer \in AN: {<<"PROPOSE", node, input>>} \subseteq res[node][3][peer]

org_spec == INSTANCE BrachaRBC WITH
    bcNode <- bcNode,
    bcValue <- input,
    predicate <- [p \in CN |-> TRUE],
    output <- [o \in CN |-> NotValue],
    msgs <- IF IsProposeSent(result, bcNode)
                THEN {[t |-> "PROPOSE", src |-> bcNode, v |-> input]}
                ELSE {}

AbsStepNext == org_spec!Broadcast
AbsStepSpec == [][AbsStepNext]_<<>>

(*
    1. The propose should be sent from the broadcaster node.
*)
Liveness ==
    \/ <>IsProposeSent(result, bcNode)

TypeOK == org_spec!TypeOK

============================================================================
