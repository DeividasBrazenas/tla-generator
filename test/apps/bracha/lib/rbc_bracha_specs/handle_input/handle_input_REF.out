@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.18 of Day Month 20?? (rev: 6dfec66)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 75 and seed 7578823342866476937 with 1 worker on 8 cores with 7220MB heap and 64MB offheap memory [pid: 25592] (Windows 11 10.0 amd64, Oracle Corporation 18.0.1.1 x86_64, MSBDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file C:\University\Master's Thesis\tla_generator_v2\test\apps\bracha\lib\rbc_bracha_specs\handle_input\handle_input_REF.tla (file:/c:/University/Master's%20Thesis/tla_generator_v2/test/apps/bracha/lib/rbc_bracha_specs/handle_input/handle_input_REF.tla)
Parsing file C:\University\Master's Thesis\tla_generator_v2\test\apps\bracha\lib\rbc_bracha_specs\handle_input\handle_input.tla (file:/c:/University/Master's%20Thesis/tla_generator_v2/test/apps/bracha/lib/rbc_bracha_specs/handle_input/handle_input.tla)
Warning: symbols were renamed.
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-501990156630456688\Naturals.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3106/tools/tla2tools.jar!/tla2sany/StandardModules/Naturals.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-501990156630456688\FiniteSets.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3106/tools/tla2tools.jar!/tla2sany/StandardModules/FiniteSets.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-501990156630456688\Sequences.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3106/tools/tla2tools.jar!/tla2sany/StandardModules/Sequences.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-501990156630456688\TLC.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3106/tools/tla2tools.jar!/tla2sany/StandardModules/TLC.tla)
Parsing file C:\University\Master's Thesis\tla_generator_v2\test\apps\bracha\lib\rbc_bracha_specs\handle_input\BrachaRBC.tla (file:/c:/University/Master's%20Thesis/tla_generator_v2/test/apps/bracha/lib/rbc_bracha_specs/handle_input/BrachaRBC.tla)
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module handle_input
Semantic processing of module BrachaRBC
Semantic processing of module handle_input_REF
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2022-12-31 11:43:23)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2212:0 @!@!@
Implied-temporal checking--satisfiability problem has 1 branches.
@!@!@ENDMSG 2212 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 2 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 4 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 7 distinct states generated at 2022-12-31 11:43:23.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(7) at 2022-12-31 11:43:23: 518 states generated, 224 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2192:0 @!@!@
Checking temporal properties for the complete state space with 224 total distinct states at (2022-12-31 11:43:23)
@!@!@ENDMSG 2192 @!@!@
@!@!@STARTMSG 2267:0 @!@!@
Finished checking temporal properties in 00s at 2022-12-31 11:43:23
@!@!@ENDMSG 2267 @!@!@
@!@!@STARTMSG 2193:0 @!@!@
Model checking completed. No error has been found.
  Estimates of the probability that TLC did not check all reachable states
  because two distinct states had the same fingerprint:
  calculated (optimistic):  val = 3.6E-15
@!@!@ENDMSG 2193 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2022-12-31 11:43:23
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 92, col 1 to line 92, col 4 of module handle_input>: 7:7
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 93, col 12 to line 93, col 24 of module handle_input: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 94, col 21 to line 108, col 35 of module handle_input: 4:16
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 94, col 34 to line 94, col 35 of module handle_input: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 109, col 23 to line 109, col 80 of module handle_input: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 109, col 28 to line 109, col 40 of module handle_input: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 109, col 45 to line 109, col 49 of module handle_input: 3
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 109, col 54 to line 109, col 66 of module handle_input: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 109, col 71 to line 109, col 80 of module handle_input: 1:2
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 109, col 72 to line 109, col 79 of module handle_input: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 110, col 19 to line 110, col 41 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 110, col 32 to line 110, col 33 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 112, col 18 to line 112, col 45 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 112, col 28 to line 112, col 29 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 113, col 26 to line 113, col 47 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 113, col 36 to line 113, col 37 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 114, col 17 to line 114, col 38 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 114, col 27 to line 114, col 28 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 115, col 21 to line 115, col 42 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 115, col 31 to line 115, col 32 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 116, col 20 to line 116, col 41 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 116, col 30 to line 116, col 31 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 117, col 27 to line 117, col 48 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 117, col 37 to line 117, col 38 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 118, col 21 to line 118, col 42 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 118, col 31 to line 118, col 32 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 119, col 12 to line 119, col 54 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 119, col 17 to line 119, col 54 of module handle_input: 7:28
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 119, col 27 to line 119, col 33 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<handle_input_ line 121, col 1 to line 121, col 19 of module handle_input>: 173:392
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 121, col 27 to line 121, col 52 of module handle_input: 1288
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 121, col 27 to line 121, col 34 of module handle_input: 896
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 122, col 27 to line 122, col 93 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 123, col 27 to line 123, col 66 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 124, col 27 to line 124, col 78 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 125, col 27 to line 125, col 75 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 126, col 27 to line 126, col 96 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 127, col 30 to line 127, col 60 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 128, col 35 to line 128, col 71 of module handle_input: 336
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 129, col 35 to line 129, col 78 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 130, col 27 to line 130, col 80 of module handle_input: 392
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<after_pin_0 line 132, col 1 to line 132, col 17 of module handle_input>: 22:56
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 132, col 25 to line 132, col 48 of module handle_input: 952
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 132, col 25 to line 132, col 32 of module handle_input: 896
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 133, col 28 to line 133, col 54 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 134, col 36 to line 134, col 69 of module handle_input: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 135, col 33 to line 135, col 76 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 136, col 25 to line 138, col 60 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<after_pin_1 line 140, col 1 to line 140, col 17 of module handle_input>: 22:56
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 140, col 25 to line 140, col 48 of module handle_input: 952
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 140, col 25 to line 140, col 32 of module handle_input: 896
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 141, col 25 to line 141, col 71 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 142, col 25 to line 142, col 113 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 142, col 33 to line 142, col 113 of module handle_input: 56:224
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 142, col 46 to line 142, col 56 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 143, col 25 to line 143, col 101 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 143, col 35 to line 143, col 101 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 143, col 36 to line 143, col 41 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 143, col 50 to line 143, col 100 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  ||line 143, col 60 to line 143, col 100 of module handle_input: 56:1512
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 143, col 68 to line 143, col 77 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 143, col 80 to line 143, col 84 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 143, col 87 to line 143, col 98 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 144, col 25 to line 144, col 58 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 145, col 25 to line 146, col 67 of module handle_input: 56
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Terminating line 152, col 1 to line 152, col 11 of module handle_input>: 0:7
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 152, col 40 to line 152, col 56 of module handle_input: 410
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 152, col 40 to line 152, col 47 of module handle_input: 382
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 152, col 31 to line 152, col 37 of module handle_input: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 153, col 19 to line 153, col 32 of module handle_input: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2774:0 @!@!@
<TypeOK line 32, col 1 to line 32, col 6 of module handle_input_REF>
@!@!@ENDMSG 2774 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 32, col 11 to line 32, col 25 of module handle_input_REF: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 14, col 1 to line 21, col 23 of module handle_input_REF: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 18, col 15 to line 18, col 37 of module handle_input_REF: 320:960
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 18, col 22 to line 18, col 23 of module handle_input_REF: 320
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 14, col 1 to line 21, col 23 of module handle_input_REF: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 15, col 15 to line 15, col 20 of module handle_input_REF: 783
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 14, col 1 to line 21, col 23 of module handle_input_REF: 302
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 14, col 1 to line 21, col 23 of module handle_input_REF: 735
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 19, col 13 to line 21, col 23 of module handle_input_REF: 1022
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 19, col 16 to line 19, col 44 of module handle_input_REF: 1022
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 10, col 5 to line 12, col 80 of module handle_input_REF: 1022
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 10, col 8 to line 10, col 24 of module handle_input_REF: 1022
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 11, col 8 to line 11, col 39 of module handle_input_REF: 182
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 12, col 8 to line 12, col 80 of module handle_input_REF: 182
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 12, col 24 to line 12, col 80 of module handle_input_REF: 728
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  ||||||line 12, col 24 to line 12, col 51 of module handle_input_REF: 728:1456
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||||line 12, col 25 to line 12, col 50 of module handle_input_REF: 728
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||||line 12, col 63 to line 12, col 80 of module handle_input_REF: 728
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 12, col 20 to line 12, col 21 of module handle_input_REF: 182
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 19, col 30 to line 19, col 35 of module handle_input_REF: 1262
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 19, col 38 to line 19, col 43 of module handle_input_REF: 1454
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  ||line 20, col 22 to line 20, col 69 of module handle_input_REF: 182:268
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 20, col 23 to line 20, col 68 of module handle_input_REF: 182
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  ||line 21, col 22 to line 21, col 23 of module handle_input_REF: 840:528
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 16, col 16 to line 16, col 20 of module handle_input_REF: 632
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 17, col 18 to line 17, col 36 of module handle_input_REF: 320:960
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 17, col 25 to line 17, col 26 of module handle_input_REF: 320
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 78, col 5 to line 83, col 28 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 78, col 8 to line 78, col 25 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 78, col 8 to line 78, col 11 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 78, col 23 to line 78, col 25 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 72, col 8 to line 72, col 57 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 72, col 9 to line 72, col 40 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 72, col 43 to line 72, col 49 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 72, col 55 to line 72, col 56 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||||line 56, col 8 to line 56, col 28 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||||line 56, col 8 to line 56, col 12 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |||||||line 56, col 19 to line 56, col 28 of module BrachaRBC: 224:448
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||||||line 56, col 20 to line 56, col 27 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 79, col 8 to line 79, col 20 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 80, col 8 to line 81, col 45 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 80, col 11 to line 80, col 44 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 80, col 11 to line 80, col 23 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 80, col 28 to line 80, col 44 of module BrachaRBC: 192
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 81, col 11 to line 81, col 45 of module BrachaRBC: 32
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 82, col 8 to line 82, col 36 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 83, col 8 to line 83, col 28 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 83, col 8 to line 83, col 13 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||line 83, col 19 to line 83, col 28 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 83, col 20 to line 83, col 21 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||line 83, col 26 to line 83, col 27 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||line 56, col 8 to line 56, col 28 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||||||line 56, col 8 to line 56, col 12 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  ||||||line 56, col 19 to line 56, col 28 of module BrachaRBC: 224:448
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |||||||line 56, col 20 to line 56, col 27 of module BrachaRBC: 224
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(7) at 2022-12-31 11:43:23: 518 states generated (31,811 s/min), 224 distinct states found (13,756 ds/min), 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
518 states generated, 224 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 7.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2268:0 @!@!@
The average outdegree of the complete state graph is 1 (minimum is 0, the maximum 4 and the 95th percentile is 3).
@!@!@ENDMSG 2268 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 984ms at (2022-12-31 11:43:23)
@!@!@ENDMSG 2186 @!@!@
